docs/resources/alert_policy.md

```diff
@@ -47,6 +47,26 @@ resource "nobl9_alert_policy" "this" {
     name = "my-alert-method"
   }
 }
+
+resource "nobl9_alert_policy" "this" {
+  name         = "${nobl9_project.this.name}-slow-burn"
+  project      = nobl9_project.this.name
+  display_name = "${nobl9_project.this.display_name} Slow burn (1x12h and 2x15min)"
```

Comment:
I think Peter mentioned that we should include in the terraform documentation both slow burn and fast burn the same way as we do in presets (here we have only slow burn), is that still relevant?

Reply:
I think I asked Peter if he mentioned new documentation (docs.nobl9.com) and he answered yes, but this is not a problem to add another example here too, why not! Maybe you are right.

I took it from here https://docs.google.com/document/d/17dQh_oOJmrfWugEa52NRJMyKkKj0C5LWCflo5aW5CYc/edit